Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -521,10 +522,14 @@ def _collect_pack(
             # collect only to upload if:
             # 1. collecting for marketplacev2 and pack is XSOAR & XSIAM - we want it to be uploaded but not installed
             # 2. allow_incompatible_marketplace=False, if True, then should be also to install
+            # 3. If changes are done to README files. Upload only.
             if self.marketplace == MarketplaceVersions.MarketplaceV2 and is_xsoar_and_xsiam_pack and \
                     not allow_incompatible_marketplace:
                 collect_only_to_upload = True
 
+            if reason == CollectionReason.README_FILE_CHANGED:
+                collect_only_to_upload = True
```

Comment:
What if the content item is compatible with the marketplace, then it will be collected also to install